bin/dist_get

```diff
@@ -13,6 +13,26 @@ check_writeable() {
 	printf "" > "$1" && rm "$1"
 }
 
+try_download() {
+	url="$1"
+	output="$2"
+	command="$3"
+	util_name=$(set -- $3; echo $1)
```

Comment:
This might be reason it fails on circle, or other bashizm. CircleCI uses fsh as sh bin. You probably should change the hashbang to bash or remove the bashizm (preferably the latter)